src/index.css

```diff
@@ -16,19 +16,26 @@
 .app__header__img {
   object-fit: contain;
   width: 150px;
+  
 }
 
 .app__header {
```

Comment:
give a white background to app header and some margin-right(approx 10px) to logout button